From a63247ae787b942bef9f6776682f37de3d48d106 Mon Sep 17 00:00:00 2001
From: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Date: Fri, 12 Aug 2011 00:57:26 +0300
Subject: [PATCH 01/38] man: Escape backslash properly in tapset::snmp

This patch fixes the following error:

$ man tapset::snmp > /dev/null
<standard input>:128: a space character is not allowed in an escape name
---
 man/tapset::snmp.3stap |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Author: Timo Juhani Lindfors <timo.lindfors@iki.fi>
Applied-Upstream: http://sources.redhat.com/git/gitweb.cgi?p=systemtap.git;a=commit;h=a63247ae787b942bef9f6776682f37de3d48d106

diff --git a/man/tapset::snmp.3stap b/man/tapset::snmp.3stap
index d6c1401..c315237 100644
--- a/man/tapset::snmp.3stap
+++ b/man/tapset::snmp.3stap
@@ -125,7 +125,7 @@ function tcpmib_filter_key: long ( sk:long, op:long ){
 /* Print the results. */
 probe end {
 	foreach (addr in  RetransSegs )
-		printf ("%s  %d \n ",ip_ntop(htonl(addr)), lport)
+		printf ("%s  %d \\n ",ip_ntop(htonl(addr)), lport)
 }
 .ESAMPLE
 	
-- 
1.7.2.5

